OneOneOne/OneOneOne/message_handler.py

```diff
@@ -0,0 +1,190 @@
+from lxml import etree, objectify
+import xml.etree.ElementTree as ET
+
+from builder.pystache_message_builder import PystacheMessageBuilder
+from utilities.file_utilities import FileUtilities
+from definitions import XML_PATH, TEMPLATE_PATH
+import logging
+
+basic_success_response = FileUtilities.get_file_string(XML_PATH / 'basic_success_response.xml')
+
+base_fault_response = FileUtilities.get_file_string(XML_PATH / 'basic_fault_response.xml')
+
+
+def build_error_message(error):
+    builder = PystacheMessageBuilder(str(TEMPLATE_PATH), 'base_error_template')
+    return builder.build_message({"errorMessage": error})
+
+
+class MessageHandler:
+    namespaces = {
+        'soap': 'http://schemas.xmlsoap.org/soap/envelope/',
+        'a': 'http://www.etis.fskab.se/v1.0/ETISws',
+        'wsa': 'http://www.w3.org/2005/08/addressing',
+        'itk': 'urn:nhs-itk:ns:201005'
+    }
+
+    def __init__(self, message_string):
+        logging.basicConfig(level=logging.DEBUG)
+        self.message = message_string
+        self.message_tree = ET.fromstring(self.message)
+
+        self.check_list = [
+            self.check_action_types,
+            self.check_manifest_and_payload_count,
+            self.check_manifest_count_against_actual,
+            self.check_payload_count_against_actual,
+            self.check_payload_id_matches_manifest_id
```

Comment:
I don't think all these in the array need to have the `self` prefix. I could be wrong. But looks like you just need access to `check_list` later on.

Reply:
Again ignore me here. I can see these are functions within this class.